src/Pipedrive.net/Models/Request/NewWebhook.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Pipedrive
+{
+    public class NewWebhook
+    {
+        [JsonProperty("subscription_url")]
+        public string SubscriptionUrl { get; set; }
+
+        [JsonProperty("event_action")]
+        public string EventAction { get; set; }
+
+        [JsonProperty("event_object")]
+        public string EventObject { get; set; }
+
+        [JsonProperty("user_id")]
+        public long? UserId { get; set; }
+
+        [JsonProperty("http_auth_user")]
+        public string HttpAuthUser { get; set; }
+
+        [JsonProperty("http_auth_password")]
+        public string HttpAuthPassword { get; set; }
+
+        [JsonIgnore]
+        private readonly List<string> _validEventActions = new List<string>
```

Comment:
Why not introduce a new enum `EventAction`? No need to have an extra list here and you could remove the check in the constructor